fews_3di/simulation.py

```diff
@@ -166,14 +168,29 @@ def run(self):
         else:
             logger.info("Saved state not enabled in the configuration, skipping.")
 
-        rain_file = self.settings.base_dir / "input" / "precipitation.nc"
-        if rain_file.exists():
-            rain_raster_netcdf = utils.write_netcdf_with_time_indexes(
-                rain_file, self.settings
-            )
-            self._add_rain(rain_raster_netcdf)
-        else:
-            logger.info("No rain file found at %s, skipping.", rain_file)
+        if self.settings.rain_type == "constant":
+            self._add_constant_rain()
+
+        # deze functie bestaat nog niet (nog niet in api ingebouwd)
+        # elif rain_type == 'design':
+        # self._add_design_rain()
+
+        elif self.settings.rain_type == "radar":
+            self._add_radar_rain()
+
+        elif self.settings.rain_type == "custom":
+            if self.settings.rain_input == "rain_netcdf":
+                rain_netcdf = self.settings.base_dir / "input" / "precipitation.nc"
```

Comment:
Er stond een keurige nette check in de code, inclusief nette foutmelding, of de `precipitation.nc` wel bestond of niet. Maar die check lijkt helemaal weggelaten te zijn. Is dat de bedoeling?